sdk/storage/Azure.ResourceManager.Storage/src/Generated/FileShareData.cs

```diff
@@ -65,7 +65,7 @@ internal FileShareData(ResourceIdentifier id, string name, ResourceType resource
             ShareUsageBytes = shareUsageBytes;
             LeaseStatus = leaseStatus;
             LeaseState = leaseState;
-            LeaseDuration = leaseDuration;
+            LeaseDurationType = leaseDurationType;
```

Comment:
The same property calls "LeaseDuration" in dataplane. We should keep terminology alignment between dataplane and management plane.
https://github.com/Azure/azure-sdk-for-net/blob/3b0aed1451566beef9f7b57158d0b4025af28496/sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareProperties.cs#L93

Reply:
The type name change should be OK.